### PR TITLE
Make eslint config publishable

### PR DESCRIPTION
**Pull request by Hotell**
- setup tools/scripts for various node scripts and helpers
- setup custom builder for eslint-config to make it publishable

### CODE REVIEW

File: workspace.json [hunk 1]
```diff
@@ -27,14 +27,17 @@
           }
         },
         "build": {
-          "builder": "@nrwl/web:bundle",
+          "builder": "@nrwl/workspace:run-commands",
```
Comment (jukben) [inline workspace.json:63]: So if I understood it correctly – this is already supported https://github.com/nrwl/nx/pull/2226? If so I would just create issue to refactor it later. I think the current solution is good enough for us now. What do you think?

**Review by jukben — APPROVED** [review]
Nice one! Once we merge it – let's publish it.

File: workspace.json (hunk 1 above)
Comment (Hotell) [reply workspace.json:63]: yup!

File: libs/eslint-config/README.md [hunk 2]
```diff
@@ -38,36 +38,6 @@ const config = {
 };
 ```
 
-### legacy
```
Comment (Hotell) [inline libs/eslint-config/README.md:41]: I updated api docs

File: tools/scripts/build-package-json.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+  copyAssets(workspaceRoot, builderOptions);
+  updatePackageJson(workspaceRoot, builderOptions);
+}
+
+/**
+ * @param {string} workspaceRoot
+ * @param {BuilderOptions} options
+ */
+function copyAssets(workspaceRoot, options) {
```
Comment (Hotell) [inline tools/scripts/build-package-json.js:70]: added missing copyAssets ( to copy markdown)

File: tools/scripts/types.d.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { CoreProperties } from '@schematics/update/update/package-json';
+
+export interface PackageJSON extends CoreProperties {
```
Comment (Hotell) [inline tools/scripts/types.d.ts:3]: used existing packagejson definition

**Review by Hotell — COMMENTED** [review]
added few tweaks, pls check it out, if all good we can rebase/merge

**Review by jukben — APPROVED** [review]

**Pull request merged**